onnx/defs/experiments/functions.cc

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
```

Comment:
let's rename this file to experiments_functions.cc. Otherwise, there will be many functions.cc (like defs.cc, which is not a good idea). It makes hard to find some definition codes. :) Thank you!